app/controllers/requests_controller.rb

```diff
@@ -58,23 +58,23 @@ def update
     if (old_status == 'outstanding' && request_params[:status] == 'approved')
       userMadeRequest = true
       # UserMailer.request_approved_email_all_subscribers(current_user, @request, userMadeRequest).deliver_now
```

Comment:
Didn't have internet when testing this so had to uncomment emails, going to fix this later